src/pages/ProfileForm/Step1Form/Step1Form.tsx

```diff
@@ -190,7 +190,7 @@ function Step1Form(): JSX.Element {
     <div id='step1form'>
       <FormWrapper isMarginTop={false}>
         <CrumbTrail>
-          <Link href='/landing'>Platform home</Link>
+          <Link href='/landing'>Home</Link>
```

Comment:
Similar to my comment above: this breadcrumb won't work, because users who hit this page can't go the landing page until they've been associated with an institution.